modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -0,0 +1,401 @@
+
+* As a major prerequisite, the Kiteworks instance *must* be running on the "Venice" release or higher. If this requirement is not met, migration can not be started.
+
+* You need to login into the Kiteworks appliance as role *System Admin*.
+
+* The Kiteworks system must provide sufficient disk space for the data to be migrated. The ownCloud xref:migration-verification[occ migrate:verify] step will report the estimated disk space needed.  
+
+* Assure sufficient quota settings in the Kiteworks user profiles.
+
+* If it is planned to integrate Kiteworks into LDAP:
++
+--
+IMPORTANT: We recommend to have the Kiteworks PCN connected and configured to an LDAP server _before_  starting the migration. This will avoid conflicting user entries that will exist in the local database additionally to the LDAP server connected.
+--
+
+* If it is planned to use a virus scanner in Kiteworks:
++
+--
+IMPORTANT: We recommend to have the Kiteworks PCN configured using a virus scanner _before_ starting the migration. This way, infected files will be put under quarantine already during migration.
+--
+
+* In the Kiteworks Admin Console, click btn:[Create Custom Application]:
++
+image:maintenance/migrate_kiteworks/kiteworks-api-settings.png[Kiteworks create a new custom application, width=300]
++
+For the settings, use the following:
+
+** Use a speaking name
+** Check btn:[Authorization Code]
+** The btn:[Access Token Lifetime] can be set to the default value.
+** Set the btn:[Redirect URI] to the default example value as shown when clicking into the field. +
+Note, the redirect URI is not used, entering the default example is therefore ok.
+* You will get a:
+** Client application ID
+** Secret key +
+Note that you only see the secret once, remember it!
+
++
+These two values are needed to initialize the xref:migration-initialization[ownCloud migration app].
+
+Finally, you have the following Kiteworks values that are needed for the next steps. We recommend to have them saved as shell variables for ease of use. In the upcoming examples, the following names represent the corresponding values:
+
+* Host name or IP address +
+`KW_HOST`
+
+* Admin users eMail address +
+`KW_ADMIN_USER`
+
+* Client application ID +
+`KW_APPLICATION_ID`
+
+* Secret key +
+`KW_SECRET`
+
+== Migration Steps
+
+After the above prerequisites have been met, the migration process can be started. The process has the following steps:
+
+* Initialization
+* Verification
+* Migration
+
+NOTE: Both the verification and migration command need the initialisation step upfront to properly communicate with the Kiteworks instance.
+
+=== Migration Initialization
+
+The migration initialization is a mandatory step and will create a json file that is used to create a so called "Satellite" - a trusted partner - on the Kiteworks instance.
+
+[source,bash]
+----
+sudo -u www-data php \
+  /var/www/owncloud/occ \
+  migrate:to-kiteworks:init \
+  $KW_HOST \
+  $KW_APPLICATION_ID \
+  $KW_SECRET
```

Comment:
```suggestion
  "$KW_SECRET"
```

Suggest to have quotes around that value. it may contain characters that are special to the shell.